doc/reference/glossary.md

```diff
@@ -16,94 +15,141 @@
 
 ## Response Object Registry
 
-A list of available `response objects`. Drives consistency in naming across `surveys` to
+A list of available `response objects`. Drives consistency in naming across `assessments` to
 support `analysis` and also to ease data entry for `Respondents`, over time.
 
-`Respondents` may add new `applications`, `data products`, and `observing systems` to the `registry`.
+`Respondents` may add new `objects` to the `registry`.
```

Comment:
@MattF-NSIDC we're thinking of changing this to `nodes` right?

Reply:
Perhaps internally in the code, as "object" has a strong meaning there. We have to loop in @hazelshapiro on this to decide if she prefers that terminology for users as well :)

Reply:
oh great question, this is another one we could get some input on or just figure out what makes sense. I'm comfortable with node or object for the code - both are intuitive.